client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,21 @@
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { AppBar, Typography } from '@material-ui/core';
+import logo from '../../Images/logo.png';
+
+const NavBar = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static" className={classes.navBar}>
+          <img className={classes.heroImage} src={logo} />
+          <div className={classes.navText}>
```

Comment:
You can use a Material UI `<Box>` component here, it also has flexbox built into it :)

Ref: https://material-ui.com/components/box/